packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -394,104 +400,57 @@ class DatePicker extends React.Component {
-          <DateInput
-            onCalendarButtonClick={this.handleOnCalendarButtonClick}
-            inputAttributes={inputAttributes}
-            isIncomplete={isIncomplete}
-            isInvalid={isInvalid}
-            required={required}
-            shouldShowPicker={!this.isDefaultDateAcceptable && this.state.selectedDate === null}
-            onButtonFocus={this.handleFocus}
-            buttonRefCallback={(buttonRef) => { this.calendarButton = buttonRef; }}
-          />
-        )}
-        excludeDates={excludeMomentDates}
-        filterDate={this.handleFilterDate}
-        includeDates={includeMomentDates}
-        maxDate={maxMomentDate}
-        minDate={minMomentDate}
-        todayButton={todayString}
-        withPortal
-        dateFormatCalendar=" "
-        dateFormat={dateFormat}
-        fixedHeight
-        locale={intl.locale}
-        placeholderText={placeholderText}
-        dropdownMode="select"
-        showMonthDropdown
-        showYearDropdown
-        preventOpenOnFocus
-        name={name}
-        allowSameDay
-      />
-    );
-
-    const popupPicker = (
-      <ReactDatePicker
-        {...customProps}
-        selected={selectedDateInPicker}
-        value={formattedValue}
-        onBlur={this.handleBlur}
-        onChange={this.handleChange}
-        onChangeRaw={this.handleChangeRaw}
-        onClickOutside={this.handleOnClickOutside}
-        onFocus={this.handleOnInputFocus}
-        onSelect={this.handleOnSelect}
-        required={required}
-        customInput={(
-          <DateInput
-            onCalendarButtonClick={this.handleOnCalendarButtonClick}
-            inputAttributes={inputAttributes}
-            isIncomplete={isIncomplete}
-            isInvalid={isInvalid}
-            shouldShowPicker={!this.isDefaultDateAcceptable && this.state.selectedDate === null}
-            onButtonFocus={this.handleFocus}
-            buttonRefCallback={(buttonRef) => { this.calendarButton = buttonRef; }}
-          />
-        )}
-        excludeDates={excludeMomentDates}
-        filterDate={this.handleFilterDate}
-        includeDates={includeMomentDates}
-        maxDate={maxMomentDate}
-        minDate={minMomentDate}
-        todayButton={todayString}
-        dateFormatCalendar=" "
-        dateFormat={dateFormat}
-        fixedHeight
-        locale={intl.locale}
-        placeholderText={placeholderText}
-        dropdownMode="select"
-        showMonthDropdown
-        showYearDropdown
-        preventOpenOnFocus
-        name={name}
-        allowSameDay
-      />
-    );
-
     return (
       <div
         className={cx('date-picker')}
         ref={this.datePickerContainer}
       >
         <ResponsiveElement
+          onChange={this.handleBreakpointChange}
           responsiveTo="window"
-          tiny={portalPicker}
-          medium={popupPicker}
-        />
+        >
+          <ReactDatePicker
+            {...customProps}
+            withPortal={this.state.showPortalPicker}
```

Comment:
The only difference between the previous portalPicker and popupPicker components was passing in `withPortal`. I updated this to utilize the showPortalPicker state.